pipeline/transformer/transformer_mets/src/main/scala/uk/ac/wellcome/platform/transformer/mets/MetsData.scala

```diff
@@ -0,0 +1,47 @@
+package uk.ac.wellcome.platform.transformer.mets
+
+import uk.ac.wellcome.models.work.internal._
+
+case class MetsData(
+  recordIdentifier: String,
+  accessCondition: Option[String]
+) {
+
+  def toWork(version: Int) =
+    // TODO: change this to UnidentifiedInvisibleWork (UnidentifiedPartialWork?)
+    UnidentifiedWork(
+      title = "??? SHOULD TITLE BE OPTIONAL ???",
+      sourceIdentifier = sourceIdentifier,
+      version = version,
+      items = List(item),
+      mergeCandidates = List(mergeCandidate),
+      otherIdentifiers = Nil,
+    )
+
+  private def item: MaybeDisplayable[Item] =
+    Unidentifiable(
+      agent = Item(
+        locations = List(
+          DigitalLocation(
+            url = "???",
```

Comment:
I _think_ [this](https://github.com/wellcometrust/catalogue/blob/f293c62d38d1b6f7b3c3894c9f10fb2ecf1eb58f/pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraLocation.scala#L32) is what the URL should look like, but better check with @jtweed